analytics/js_tracking_api.rst

```diff
@@ -435,16 +437,16 @@ To track all visible content impressions you can use ``trackVisibleContentImpres
 
 Code::
 
-    _paq.push(["trackVisibleContentImpressions", checkOnScroll, timeIntervalInMs]);
+    _paq.push(["trackVisibleContentImpressions", checkOnScroll, watchInterval]);
 
 .. data:: checkOnScroll
 
-    **Required** If set to true it will invoke this function to track new visible content impressions on scroll event.
+    **Required** ``boolean`` If set to true it will invoke this function to track new visible content impressions on scroll event.
     (It won't detect content blocks placed in a scrollable element)
 
-.. data:: timeIntervalInMs
+.. data:: watchInterval
 
-    **Optional** If set it will invoke this function to track new visible content impressions on every X miliseconds. By default it is set to 750ms.
+    **Optional** ``number`` If set it will invoke this function to track new visible content impressions on every X miliseconds. By default it is set to 750ms.
```

Comment:
This phrasing suggests that this value can be not set. Is it possible to disable interval checks and base it only on scroll? What value should be send to do that?